web/app/scripts/views/map/layers-controller.js

```diff
@@ -771,7 +781,7 @@
         }
 
         function getTilekeys() {
-            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams()).then(
+            var primary = QueryBuilder.djangoQuery(0, getAdditionalParams(), true, true, false).then(
```

Comment:
Windshaft interactivity was pulling all the record data into the UTFGrid response, although the data JSON doesn't actually get used any more (just the date and UUID, which are separate fields), so that was an easy fix.